packages/ui/src/styles/tailwind/index.css

```diff
@@ -1,6 +1,7 @@
 @layer theme, base, components, utilities;
+
 @import "tailwindcss/theme.css" layer(theme);
-@import "tailwindcss/utilities.css" layer(utilities);
+@import "tailwindcss/utilities.css" layer(utilities) source("../../../../");
```

Comment:
Without this, tailwind wasn't detecting styles outside of the ui package ([docs](https://tailwindcss.com/docs/detecting-classes-in-source-files#setting-your-base-path))